### PR TITLE
remove vl-ycrv from live docs while its not deployed

### PR DESCRIPTION
**Pull request by MarcoWorms**

### CODE REVIEW

File: docs/getting-started/products/ycrv/overview.md [hunk 1]
```diff
@@ -26,11 +26,10 @@ Yearn passes all benefits of its veCRV position on to yCRV users who hold one of
 
 - [**st-yCRV (Staking Rewards):**](#staked-ycrv) [yVault](https://medium.com/iearn/yearn-finance-explained-what-are-vaults-and-strategies-96970560432) that receives admin fees and bribes from locked CRV.
 - [**lp-yCRV (Liquidity Pool Rewards):**](#lpd-ycrvcrv) yVault for CRV/yCRV LP tokens, autocompounds emissions and fees.
-- [**vl-yCRV (Curve Gauges Voting Power):**](#voting-with-ycrv) Vote power mechanism to vote on Curve gauge weights.
 
 Users can choose to move between activated-tokens at any time depending on which benefits they want, except for vl-yCRV which has time lock [restrictions](#voting-with-ycrv) varying from 14 to 28 days.
```
Comment (wavey0x) [inline docs/getting-started/products/ycrv/overview.md:30]: reference here that can be removed.

File: docs/getting-started/products/ycrv/overview.md [hunk 2]
```diff
@@ -26,11 +26,10 @@ Yearn passes all benefits of its veCRV position on to yCRV users who hold one of
 
 - [**st-yCRV (Staking Rewards):**](#staked-ycrv) [yVault](https://medium.com/iearn/yearn-finance-explained-what-are-vaults-and-strategies-96970560432) that receives admin fees and bribes from locked CRV.
 - [**lp-yCRV (Liquidity Pool Rewards):**](#lpd-ycrvcrv) yVault for CRV/yCRV LP tokens, autocompounds emissions and fees.
-- [**vl-yCRV (Curve Gauges Voting Power):**](#voting-with-ycrv) Vote power mechanism to vote on Curve gauge weights.
```
Comment (wavey0x) [inline docs/getting-started/products/ycrv/overview.md:29]: maybe worth adding a note here like,
"Note: a third activated token, vl-yCRV, was never rolled out due to lack of demand."

File: docs/getting-started/products/ycrv/overview.md [hunk 3]
```diff
@@ -26,11 +26,10 @@ Yearn passes all benefits of its veCRV position on to yCRV users who hold one of
...
 
-Users can choose to move between activated-tokens at any time depending on which benefits they want, except for vl-yCRV which has time lock [restrictions](#voting-with-ycrv) varying from 14 to 28 days.
+> Note: a third activated token, vl-yCRV, was never rolled out due to lack of demand.
```
Comment (MarcoWorms) [inline docs/getting-started/products/ycrv/overview.md:30]:
```suggestion
> Note: a third activated token, vl-yCRV, was planned but not rolled out due to a lack of demand.
```

**Review by 0xBasically — APPROVED** [review]
lgtm

**Pull request merged**